clis/blob-propagation-jobs-cli/src/Context.ts

```diff
@@ -16,8 +17,9 @@ export class Context {
   #finalizerQueue: Queue<BlobPropagationJobData>;
   #propagatorFlowProducer: FlowProducer;
 
-  constructor(storages: $Enums.BlobStorage[], connection: ConnectionOptions) {
+  constructor(storages: $Enums.BlobStorage[], redisUri: string) {
     const uniqueStorageNames = [...new Set(storages)];
+    const connection = new IORedis(redisUri, { maxRetriesPerRequest: null });
```

Comment:
Note we are setting `maxRetriesPerRequest` as `null`. To prevent this warning: https://github.com/nocodb/nocodb/issues/2452